amundsen_application/static/js/components/common/Badges/styles.scss

```diff
@@ -0,0 +1,12 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+
+.clickable-badge-primary {
```

Comment:
This only applies the style to badges with the primary style, but we will want it applied to all badges all of the time. So you can either:
1. Use the suggestion from my previous comment that will apply this to any `.clickable-badge-xyz` classes.
2. Update the component to split the classNames such that every badge has the `clickable-badge` class + a class that defines the labelStyle, and update this style here to `.clickable-badge`. 
e.g. ` className={'clickable-badge ${this.props.labelStyle}'}`

Reply:
Yeah until I implement the style changes there is no reason to have a distinction here so I will just change it to clickable-badge, I can use your suggestion for the distinction on my next PR